R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -3921,14 +3921,14 @@ test_that("No extra files are created in SPARK_HOME by starting session and maki
   # before creating a SparkSession with enableHiveSupport = T at the top of this test file
   # (filesBefore). The test here is to compare that (filesBefore) against the list of files before
   # any test is run in run-all.R (sparkRFilesBefore).
-  # sparkRWhitelistSQLDirs is also defined in run-all.R, and should contain only 2 whitelisted dirs,
+  # sparkRIncludedSQLDirs is also defined in run-all.R, and should contain only 2 included dirs,
```

Comment:
does it make sense to have `allowed` here instead of `included`?

Reply:
Yup, this is more appropriate. Thanks.